cctrl/auth.py

```diff
@@ -92,12 +132,73 @@ def delete_tokenfile(settings):
     return False
 
 
+def set_configfile(settings, email=None, ssh_auth=None, ssh_path=None):
+    create_config_dir(settings)
+    config = ConfigParser.ConfigParser()
+    config.read(settings.config_path)
+
+    if not config.has_section('user'):
+        config.add_section('user')
+
+    if email:
+        config.set('user', 'email', email)
+
+    if ssh_auth is not None:
```

Comment:
Because `ssh_auth` is a boolean value and we check that it's not set at all, rather that it's `True`.